test/IntegrationTests/Helpers/MockMetricsCollector.cs

```diff
@@ -46,7 +44,7 @@ public class MockMetricsCollector : IDisposable
     private readonly List<Expectation> _expectations = new();
     private readonly BlockingCollection<List<Collected>> _metricsSnapshots = new(10); // bounded to avoid memory leak; contains protobuf type
 
-    private MockMetricsCollector(ITestOutputHelper output, string host = "localhost")
+    public MockMetricsCollector(ITestOutputHelper output, string host = "localhost")
```

Comment:
There is an issue left by me, the host is only available for .NET Fx
If not now, then maybe for some future refactor.

Reply:
Yeah I saw it and I was even thinking if I should address it... But I decided that is not needed now so omitted it. AFAIK it is needed only for `AspNetTests`